esy.json

```diff
@@ -9,14 +9,19 @@
   },
   "esy": {
     "build": [
-      [ "node", "scripts/install.js" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'bsconfig.json'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'esy.json'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'jscomp'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'lib'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / '.merlin'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'scripts'}", "#{self.install}" ],
-      [ "cp", "-r", "-f", "#{self.target_dir / 'vendor'}", "#{self.install}" ]
+      ["node", "scripts/install.js"],
```

Comment:
Apologies. Only relevant line here is the removal of .merlin file from the list of files being copied